guides/gitconfig.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Managing Git Configuration
+description: Learn how to manage your Git configuration in Coder.
+---
+
+This guide will show you how to manage Git configuration through the system,
+global, and repo levels.
+
+## How to use the System and Global configurations
+
+For customers seeking organizational (system-level) `.gitconfig` defaults, we recommend
+adding a default configuration into the `/etc/gitconfig` file of each image associated
+to the organization:
```

Comment:
I think it may be useful to give context for System configurations. Furthermore, we don't offer any concrete advice on how to set the file.

Here's a few suggestions:

1. Start with a context of git configurations that apply to an organization

```
For scenarios whereby a set of git configurations should apply across an organization,
we recommend using a System-level git configuration. System-level git configurations
are located at `/etc/gitconfig`. These settings are applied to each git repository, but may
be overriden using global or worktree git configurations.
```

2. There are various ways to apply `/etc/gitconfig`. We suggest `coder/configure` for
applying these settings to a development image. Otherwise, they may be applied
using `~/personalize` or some other mechanism such as an install script or file sync (we don't need to document such).

Reply:
thanks for the feedback @vapurrmaid - I can certainly add the context you provided above. I think that will be helpful in framing the guide for the user. 

Question - where can one edit `coder/configure` to apply System-level git defaults?

Reply:
I was wrong about this comment - the recommended way to add a system gitconfig would be directly in the Dockerfile:

```dockerfile
# Add system level gitconfig
COPY [".gitconfig", "/etc/gitconfig"]
```

In fact, we do this ourselves.

I'm sorry for steering in the wrong direction with `coder/configure`, which is for more complicated needs.

For more information on `coder/configure`, we can easily refer to <https://coder.com/docs/images/configure>

Reply:
No worries - appreciate the clarification. I'll push another commit to clarify this and add the manifest file.